data/themes/darktable.css

```diff
@@ -908,6 +908,13 @@ dialog *
   padding:0;
 }
 
+dialog headerbar
+{
+  background-color: @border_color;
+  margin: 0;
+  padding: 1pt;
```

Comment:
Adding 1pt for padding works for the top and bottom of the header. However, there isn't enough padding for the sides. Looking elsewhere in the CSS file, I see the value of 0.5em used for element padding, so I'd suggest the following:

```suggestion
  padding: 1pt 0.5em;
```

There is still the issue with the buttons in the header having no padding for them, but that's (probably) a different bug / different PR, unless you think it's suitable here too.

Reply:
I've replace your suggestion with :
`padding: 0.2em 0.6em`
That seems better for alignement and spacing on my config.

Reply:
I will commit it if it's ok for you :

![Capture-20190504155748-994x86](https://user-images.githubusercontent.com/45535283/57180082-67a3bb00-6e85-11e9-91a3-70780d45e0da.png)

Reply:
Yeah, that looks good from what I see in the screenshot. :+1:

I was wondering why `pt` was used, as it doesn't scale* with the font, unlike `em`, which is tied to the font size. Good to see it in `em` here. Perhaps it makes sense to review & generalize the units used across the CSS, possibly with variables — but that's a different conversation outside the scope of this PR. :wink:

(* At least this is with web CSS — and I'm guessing it's probably the case with GTK CSS too?)

Reply:
I don't know why pt was used as it was the original work of @aurelienpierre.

I also improve buttons just now with this add :

`
dialog headerbar button
{
  padding: 1px 2px;
}
`
If it concerns improvement css theme, I think this PR could be also for other concerns. If needed I could change title.

I've think about change all scales and delete pt but it could also break something. I don't know, I will check more. W3C recommands use of em and px for screen viewing.

With that said, I'm not for changing units in px by now as it works. Using em could break something as users could use another font for example.